content/docs/start/metrics-parameters-plots.md

```diff
@@ -0,0 +1,257 @@
+> Please refer to `dvc metrics` and `dvc plots` for more details.
+
+Let's save this iteration, so we can compare it later:
+
+```dvc
+$ git add scores.json prc.json roc.json
+$ git commit -a -m "Create evaluation stage"
+```
+
+Later we will see how these and other can be used to compare and visualize
+different pipeline iterations. For now, let's see how can we capture another
+important piece of information that will be useful for comparison: parameters.
+
+## Defining parameters
+
+It's pretty common for data science pipelines to include configuration files
+that define adjustable parameters to train a model, do pre-processing, etc. DVC
+provides a mechanism for stages to depend on the values of specific sections of
+such a config file (YAML, JSON, TOML, and Python formats are supported).
+
+Luckily, we should already have a stage with
+[parameters](/doc/command-reference/params) in `dvc.yaml`:
+
+```yaml
+featurize:
+  cmd: python src/featurization.py data/prepared data/features
+  deps:
+    - data/prepared
+    - src/featurization.py
+  params:
+    - featurize.max_features
+    - featurize.ngrams
+  outs:
+    - data/features
+```
+
+<details>
+
+### 💡 Expand to recall how it was generated.
+
+The `featurize` stage
+[was created](/doc/start/data-pipelines#dependency-graphs-dags) with this
+`dvc run` command. Notice the argument sent to the `-p` option (short for
+`--params`):
+
+```dvc
+$ dvc run -n featurize \
+          -p featurize.max_features,featurize.ngrams \
+          -d src/featurization.py -d data/prepared \
+          -o data/features \
+          python src/featurization.py data/prepared data/features
+```
+
+</details>
+
+The `params` section defines the [parameter](/doc/command-reference/params)
+dependencies of the `featurize` stage. By default DVC reads those values
+(`featurize.max_features` and `featurize.ngrams`) from a `params.yaml` file. But
+as with metrics and plots, parameter file names and structure can also be user
+and case defined.
+
+This is how our `params.yaml` file looks like:
+
+```yaml
+prepare:
+  split: 0.20
+  seed: 20170428
+
+featurize:
+  max_features: 500
+  ngrams: 1
+
+train:
+  seed: 20170428
+  n_est: 50
+  min_split: 2
+```
+
+## Updating and running
+
+Let's edit the `params.yaml` file to use bigrams and increase the number of
+features:
+
+```diff
+ featurize:
+-  max_features: 500
+-  ngrams: 1
++  max_features: 1500
++  ngrams: 2
+```
+
+And the beauty of `dvc.yaml` is that all you need to do now is to run:
+
+```dvc
+$ dvc repro
+```
+
+It'll analyze the changes, use existing cache of previous runs, and execute only
+the commands that are needed to get the new results (model, metrics, plots).
+
+The same logic applies to other possible adjustments — edit source code, update
+datasets — you do the changes, use `dvc repro`, and DVC runs what needs to be
+run.
+
+## Comparing changes
+
+Finally, let's see how the updates improved performance. DVC has a few commands
```

Comment:
changes -> versions? attempts? iterations? (in the section title)

Reply:
Assuming we use "iterating" in the previous section title, I like "iterations" here.